_docker/drupal/drupal-filesystem/web/themes/custom/rhdp2/rhd-frontend/src/styles/rhd-theme/components/_rhd-c-two-col-hero-layout.scss

```diff
@@ -94,7 +97,9 @@ $pf-global--breakpoint--temp: 1360px;
       padding-left:  var(--rhd-theme--container-spacer-md);
       padding-right:  var(--rhd-theme--container-spacer-md);
     }
+  }
 
+  .field--name-field-column-2 {
```

Comment:
Does this read that `.assembly-type-static_content_band` is only supported in the column 2 field?

Reply:
Yeah, that was what @alistairmc had originally written, I just moved the padding stuff up to apply to both columns. I'm fine with adding the column-1 to that whole block, I'm just not sure if that was done on purpose.

Reply:
If I remember, it was to make the nested components work. happy if it's been tested to work ok

Reply:
@staceymosier resolved with #428f050